Assets/CrosshairImage.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CrosshairImage : MonoBehaviour {
+
+	private float radiusL;
+	private float radiusR;
+
+	[SerializeField]
+	private GameObject[] crosshairs;// 4 : RCL's , 5 : middlecross , 6:RCL middlecross
+
+	[SerializeField]
+	private RectTransform[] crosshairsL0;
+	[SerializeField]
+	private RectTransform[] crosshairsL1;
+	[SerializeField]
+	private RectTransform[] crosshairsR0;
+	[SerializeField]
+	private RectTransform[] crosshairsR1;
+
+
+	public bool noCrosshairL = false;
+	public bool noCrosshairR = false;
+
+	public void SetRadius(float setRadiusL, float setRadiusR){
+		radiusL = setRadiusL * 25f;
+		radiusR = setRadiusR * 25f;
+
+		crosshairsL0 [0].offsetMin = new Vector2 (-radiusL, radiusL);
+		crosshairsL0 [0].offsetMax = new Vector2 (-radiusL, radiusL);
+
+		crosshairsL0 [1].offsetMin = new Vector2 (radiusL, radiusL);
+		crosshairsL0 [1].offsetMax = new Vector2 (radiusL, radiusL);
+
+		crosshairsL0 [2].offsetMin = new Vector2 (-radiusL, -radiusL);
+		crosshairsL0 [2].offsetMax = new Vector2 (-radiusL, -radiusL);
+
+		crosshairsL0 [3].offsetMin = new Vector2 (radiusL, -radiusL);
+		crosshairsL0 [3].offsetMax = new Vector2 (radiusL, -radiusL);
+
+		crosshairsL1 [0].offsetMin = new Vector2 (-radiusL, radiusL);
+		crosshairsL1 [0].offsetMax = new Vector2 (-radiusL, radiusL);
+
+		crosshairsL1 [1].offsetMin = new Vector2 (radiusL, radiusL);
+		crosshairsL1 [1].offsetMax = new Vector2 (radiusL, radiusL);
+
+		crosshairsL1 [2].offsetMin = new Vector2 (-radiusL, -radiusL);
+		crosshairsL1 [2].offsetMax = new Vector2 (-radiusL, -radiusL);
+
+		crosshairsL1 [3].offsetMin = new Vector2 (radiusL, -radiusL);
+		crosshairsL1 [3].offsetMax = new Vector2 (radiusL, -radiusL);
+
+		//R
+		crosshairsR0 [0].offsetMin = new Vector2 (-radiusR, radiusR);
+		crosshairsR0 [0].offsetMax = new Vector2 (-radiusR, radiusR);
+
+		crosshairsR0 [1].offsetMin = new Vector2 (radiusR, radiusR);
+		crosshairsR0 [1].offsetMax = new Vector2 (radiusR, radiusR);
+
+		crosshairsR0 [2].offsetMin = new Vector2 (-radiusR, -radiusR);
+		crosshairsR0 [2].offsetMax = new Vector2 (-radiusR, -radiusR);
+
+		crosshairsR0 [3].offsetMin = new Vector2 (radiusR, -radiusR);
+		crosshairsR0 [3].offsetMax = new Vector2 (radiusR, -radiusR);
+
+		crosshairsR1 [0].offsetMin = new Vector2 (-radiusR, radiusR);
+		crosshairsR1 [0].offsetMax = new Vector2 (-radiusR, radiusR);
+
+		crosshairsR1 [1].offsetMin = new Vector2 (radiusR, radiusR);
+		crosshairsR1 [1].offsetMax = new Vector2 (radiusR, radiusR);
+
+		crosshairsR1 [2].offsetMin = new Vector2 (-radiusR, -radiusR);
+		crosshairsR1 [2].offsetMax = new Vector2 (-radiusR, -radiusR);
+
+		crosshairsR1 [3].offsetMin = new Vector2 (radiusR, -radiusR);
+		crosshairsR1 [3].offsetMax = new Vector2 (radiusR, -radiusR);
+
+	}
+
+	public void SetCurrentLImage(int CurImage){
+		if(CurImage == 1){
```

Comment:
var = CurImage == 1;
crosshairs[0].SetActive(var);
crosshairs[1].SetActive(!var);

Please also add comment to explain what crosshairs[x] means in the variable declaration